apps/api/src/app/app.module.ts

```diff
@@ -250,6 +251,7 @@ const autoSchemaFile = environment.production
       load: [
         AdrAndMachineLicenseClientConfig,
         AssetsClientConfig,
+        FirearmLicenseClientConfig,
```

Comment:
Since the client returns more than just the license (firearm ownership at least, also some application endpoints which we probably wont use?), should it have a more generic name?